pkg/mads/v1/service/http.go

```diff
@@ -84,14 +86,15 @@ func (s *service) handleDiscovery(req *restful.Request, res *restful.Response) {
 		return
 	}
 
-	marshaller := &jsonpb.Marshaler{OrigName: true}
-	resStr, err := marshaller.MarshalToString(discoveryRes)
+	// TODO is this equivalent?
```

Comment:
Is it?

Reply:
Yes!

https://github.com/golang/protobuf/blob/v1.5.2/jsonpb/encode.go#L99

Reply:
Yes!

https://github.com/golang/protobuf/blob/v1.5.2/jsonpb/encode.go#L99

Reply:
but actually I'm going to move this back to `jsonpb` for consistency.